ninja-core/src/test/java/ninja/utils/NinjaPropertiesImplTest.java

```diff
@@ -43,24 +41,19 @@ public void tearDown() {
     public void testSkippingThroughModesWorks() {
 
         // check that mode tests works:
-        NinjaPropertiesImpl ninjaPropertiesImpl = new NinjaPropertiesImpl(NinjaMode.test);
```

Comment:
Nothing hass changed. I just moved from hamcrest to AssertJ + added one test at the end...